discovery/local/src/main/java/org/apache/aries/rsa/discovery/local/package-info.java

```diff
@@ -0,0 +1,2 @@
+
+package org.apache.aries.rsa.discovery.local;
```

Comment:
This file does not contain licence info. Is it really necessary to have this file?

Reply:
Not yet .. but I plan to place the capability there.